src/app/home/home.component.html

```diff
@@ -1,8 +1,56 @@
-<div class="body">
+
+
+<div *ngIf="!user" class="body">
     Welcome to the Character builder! 
     This app is designed to walk new D&D player through the processs of creating a character. 
-    Log in to create a new character and view your existing ones.
+    Sign in with you Google account to create a new character and view your existing ones.
 </div>
+
+<div class="body">
+    <div *ngIf="!user" class="card text-center">
```

Comment:
Since these ngifs and body classes are the same, can combine into something like 

```
<div class="body">
  <ng-container *ngIf="!user; else userLogin">
    <p class="welcome-message">
      Welcome to the Character builder!...
    </p>
    <div class="card text-center">
      <asl-google-signin-button type="standard"></asl-google-signin-button>
    </div>
  </ng-container>
  <ng-template #userLogin>
  ...
  </ng-template>
```